prototype2/efu/CMakeLists.txt

```diff
@@ -21,7 +21,6 @@ set(efu_INC
   Server.h
   )
   
-add_definitions(-std=c++11)
```

Comment:
This should not be needed. Already taken care of in `cmake/CompilerConfig.cmake`

Reply:
Ok.

Reply:
Actually, modern versions of cmake has other ways of defining which C++ version to use. Seeing as we are nowadays using CMake  versions > 3.0, we should probably consider modernising the CMake code a bit.